src/utils.cc

```diff
@@ -39,7 +52,12 @@ bool minio::utils::GetEnv(std::string& var, const char* name) {
 std::string minio::utils::GetHomeDir() {
   std::string home;
   if (GetEnv(home, "HOME")) return home;
+  if (GetEnv(home, "USERPROFILE")) return home;
```

Comment:
Isn't it MS Windows specific?

Reply:
yeah, it's unlikely to be set on linux but it would be safer to move it behind a preprocessor conditional